includes/im_user_dashboard.inc

```diff
@@ -224,24 +222,24 @@ function im_user_dashboard_process_action($form_state, $action) {
 	      /*Add the Regional user and save the Regional nid and regional stores to the user profile*/
  	      if ($role == "content_manager_action_regional") {
  	      	if (!empty($_SESSION["dashboard_regional_moderator_users"][$uid]['profile_nids'])) {
-		      $rmp_nodes = explode(',',$_SESSION["dashboard_regional_moderator_users"][$uid]['profile_nids']);
+		      $rmp_nodes = explode(',', $_SESSION["dashboard_regional_moderator_users"][$uid]['profile_nids']);
  	  	    	$rmp_node_stores = '';
- 	  	    	foreach($rmp_nodes as $rmp_node_ids) {
+ 	  	    	foreach ($rmp_nodes as $rmp_node_ids) {
  	  	    		$rmp_store_details = node_load($rmp_node_ids);
- 	  	    		if(!empty($rmp_store_details->field_rmp_store['und'][0]['value'])) {
- 	  	    			$rmp_node_stores.= $rmp_store_details->field_rmp_store['und'][0]['value'].',';
+ 	  	    		if (!empty($rmp_store_details->field_rmp_store['LANGUAGE_NONE'][0]['value'])) {
```

Comment:
I would remove the quotes in LANGUAGE_NONE  key mentioned and re check the same and push the code.